components/core/src/clp/ffi/ir_stream/search/QueryHandlerImpl.cpp

```
@@ -0,0 +1,324 @@
+            std::move(query),
+            std::move(auto_gen_namespace_partial_resolutions),
+            std::move(user_gen_namespace_partial_resolutions),
+            std::move(projected_columns),
+            std::move(projected_column_to_original_key),
+            case_sensitive_match
+    };
+}
+
+// TODO: Fix clang-tidy
+// NOLINTNEXTLINE(*)
+auto QueryHandlerImpl::evaluate_node_id_value_pairs(
```

Cppcheck (2.10-2)

[style] The function 'evaluate_node_id_value_pairs' is never used.

(unusedFunction)